src/java/us/kbase/mobu/compiler/TemplateBasedGenerator.java

```diff
@@ -129,26 +125,6 @@ public static void generate(List<KbService> srvs, String defaultUrl,
             rClient.close();
         }
         //////////////////////////////////////// Servers /////////////////////////////////////////
-        if (asyncByDefault) {
-            // Make all methods async and mark methods being async already as couldn't be sync
-            context.put("any_async", true);
-            List<Map<String, Object>> modules = (List<Map<String, Object>>)context.get("modules");
-            for (int modulePos = 0; modulePos < modules.size(); modulePos++) {
-                Map<String, Object> module = modules.get(modulePos);
-                module.put("any_async", true);
-                List<Map<String, Object>> methods = (List<Map<String, Object>>)module.get("methods");
-                if (methods == null)
-                    continue;
-                for (int methodPos = 0; methodPos < methods.size(); methodPos++) {
-                    Map<String, Object> method = methods.get(methodPos);
-                    //TODO ROMAN should async go into the map or can this line be deleted?
-//                    Boolean async = (Boolean)method.get("async");
-                    method.put("async", true);
-                    //if (async == null || !async)
-                    //    method.put("could_be_sync", true);
-                }
-            }
-        }
```

Comment:
This section has no effect, since none of the keys it adds to the map are used in the python server or impl files. Not sure about the other languages but they're being removed anyway